torch/csrc/jit/passes/loop_unrolling.cpp

```diff
@@ -162,10 +162,11 @@ void replaceLoopCounter(Node* loop) {
   body->insertOutput(1, result);
 }
 
-void unroll(Node* loop) {
+void unroll(Node* loop, bool constant_only) {
```

Comment:
Add a comment on what `constant_only` means.

[Readability] "constant_only" => "ignore_size_limit_for_constant_loop" (or something to that effect, I just feel that "constant_only" is misleading as we are unrolling both small-size loops and constant loops)